layouts/partials/article/title.html

```diff
@@ -5,7 +5,7 @@
 {{/*  No title - For when the main title is the same as the first article  */}}
 {{ else }}
     <div class="article-title article-actions" data-align="center-left">
-        {{ if .Data.Pages }}
+        {{ if and (not .Data.Pages) (not .Parent) }}
```

Comment:
Isn't it supposed to check if the article doesn't have a parent and has child articles e.g `if and (.Data.Pages) (not .Parent)`

Reply:
You are absolutely correct. Thanks for catching that. Have updated 
<img width="1430" alt="image" src="https://github.com/user-attachments/assets/bc4b8b95-3278-4f1c-8478-4f78bc44f055">